### DISCUSSION TITLE
Textbox Left padding

### DISCUSSION THREAD

**Discussion by fpauk** [fabricjs/fabric.js#10324] [Q&A]
Is there a way to only set Left padding?

**Comment by asturur** [comment]
No. What would be the reason for that to have an off centered bounding box for controls?

**Reply by fpauk** [reply]
Well - As L and R padding in css help makeing stuff nice., it would be great haveing it seperate too
![Fabric-padding](https://github.com/user-attachments/assets/c2eb155b-497d-4d11-8c5c-8259edbf7b44)

Regards

**Reply by asturur** [reply]
The padding property is for the colors, not for layouting text inside its own bounding box. We don't have padding at all for this usecase.
